tools/CompileReleaseNotes.py

```diff
@@ -182,7 +195,10 @@ def format_list_of_prs(prs):
         for group, prs_iterator in grouped_prs:
             group_header = {
                 "new feature": "New features",
+                "documentation": "Documentation",
                 "bugfix": "Bugfixes",
+                "build system": "Build system",
+                "ci/cd": "CI/CD",
                 None: "General changes",
```

Comment:
Do we need to add `CLI/Pybindings` here?

Reply:
Oh I just forgot to replace this dict with the one above, thanks for catching.